RedHat-UX/red-hat-design-system
# docs: update Get Started: Overview page

## What I did

Replaced old content on Get Started: Overview page with more general content and links to role-specific pages


## Testing Instructions

Check [Overview](https://deploy-preview-1435--red-hat-design-system.netlify.app/get-started/) page in DP

## Notes to Reviewers
